client/valor/client.py

```diff
@@ -273,36 +273,36 @@ def _requests_wrapper(
 
         return resp
 
-    def _requests_post_rel_host(self, endpoint: str, *args, **kwargs):
+    def _requests_post_rel_host(self, endpoint: str, *_, **kwargs):
         """
         Helper for handling POST requests.
         """
         return self._requests_wrapper(
-            method_name="post", endpoint=endpoint, *args, **kwargs
+            method_name="post", endpoint=endpoint, **kwargs
         )
 
-    def _requests_get_rel_host(self, endpoint: str, *args, **kwargs):
+    def _requests_get_rel_host(self, endpoint: str, *_, **kwargs):
```

Comment:
can you explain these "*args" -> "*_" changes a bit more? is this related to `validate_matching_label_keys` somehow?

Reply:
it seems like your client and integration changes are unrelated to this PR, and may be leaked here from another PR. just want to confirm that's true. best practice would be to remove them, but I'd consider it a nit

Reply:
Removed! They got dragged in by an unrelated pr.